src/bin/vaults/CommandPull.ts

```diff
@@ -9,12 +9,19 @@ class CommandPull extends CommandPolykey {
     super(...args);
     this.name('pull');
     this.description('Pull a Vault from Another Node');
-    this.argument('<nodeId>', 'Id of the node to pull the vault from');
-    this.argument('<vaultName>', 'Name of the vault to be pulled');
+    this.option(
+      '-ni, --node-id [nodeId]',
+      'Id of the node to pull the vault from',
+    );
+    this.option(
+      '-pv, --pull-vault [pullVaultNameOrId]',
+      'Name or Id of the vault to pull from',
+    );
```

Comment:
Move these into `bin/utils/options.ts`. They should be standardised in our bin.

Reply:
Fixed.